SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -8,16 +8,26 @@
 %define config_source %{SOURCE1}
 %endif
 
+%ifarch aarch64
+%global __provides_exclude_from %{_libdir}/debug/.build-id/
+%define arch arm64
+%define archdir arm64
+%define config_source %{SOURCE2}
+%endif
+
+# Generated by update script
+%global kernelver 5.15.98.mshv1
+
 Summary:        Linux Kernel for Kata UVM
 Name:           kernel-uvm
-Version:        5.15.48.1
-Release:        9%{?dist}
+Version:        %{kernelver}
+Release:        1%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Kernel
 URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
-Source0:        https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-2/%{version}.tar.gz#/kernel-%{version}.tar.gz
+Source0:       %{_mariner_sources_url}/kernel-mshv-%{kernelver}.tar.gz
 Source1:        config
```

Comment:
Shouldn't this point to upstream rather than our copy?

Reply:
Based on https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/kernel-mshv/kernel-mshv.spec#L20. I think this is the correct location because, if I understand, upstream now is https://microsoft.visualstudio.com/DefaultCollection/LSG/_git/linux-dom0, but we need to give a publicly accessible link for download

Reply:
Nvm, didn't see the comment above